module/VuFind/src/VuFind/Search/Base/Options.php

```diff
@@ -982,4 +982,23 @@ protected function configureAutocomplete(Config $searchSettings = null)
         $this->autocompleteAutoSubmit = $searchSettings->Autocomplete->auto_submit
             ?? $this->autocompleteAutoSubmit;
     }
+
+    /**
+     * Get advanced search limits that override the natural sorting to
+     * display at the top.
+     *
+     * @param string $limit advanced search limit
+     *
+     * @return array
+     */
+    public function limitOrderOverride($limit)
+    {
+        $facetSettings = $this->configLoader->get($this->getFacetsIni());
+        $limits = $facetSettings->Advanced_Settings->limitOrderOverride;
+        $delimiter = $facetSettings->Advanced_Settings->limitDelimiter ?
+            $facetSettings->Advanced_Settings->limitDelimiter : '::';
+        $limitConf = $limits && $limits->get($limit) ?
+            preg_replace('/\s+/', '', $limits->get($limit)) : '';
```

Comment:
What is the purpose of this preg_replace? It seems to be preventing facet values containing spaces from being priorities (I tried, for example, to move "Conference Proceeding" to the top of the format list, but it doesn't work).

Reply:
As an alternative, how would you feel about something like:

<pre>
$limitConf = $limits ? $limits->get($limit) : '';
return array_map('trim', explode($delimiter, $limitConf));
</pre>

Reply:
Good catch! Yes, the purpose was just to trim. This was actually pretty sloppy now that you point it out. I've added the patch.

Reply:
Thanks!